docker/data/logstash/env2yaml/env2yaml

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Execute Java env2yaml using Logstash's bundled JDK and SnakeYAML
+# The .class file is compiled during Docker build and placed in /usr/local/bin
+
+exec /usr/share/logstash/jdk/bin/java \
+    -cp "/usr/share/logstash/logstash-core/lib/jars/*:/usr/local/bin" \
```

Comment:
I have tested a series of JVM flags to make startup faster. overall I can see ever-so-slight benefits with `-XX:+UseSerialGC -Xms16m -Xmx16m`, I tried messing with tiered complation settings but no obvious benefit. best tests are likely run on a very limited container.